cfgov/unprocessed/css/forms.less

```diff
@@ -606,26 +608,51 @@
     margin-bottom: unit(30px / @base-font-size-px, em);
 }
 
+.form-label-header + .form-l .form-label-header {
+    // Override heading-5() settings.
+    .webfont-regular();
+    font-weight: normal;
```

Comment:
Looks like it! Fixed in https://github.com/cfpb/cfgov-refresh/commit/d0bd47229a4c3b4b8cb2f1400c6924a3fc7e5e0d